.github/workflows/update-helm.yaml

```diff
@@ -0,0 +1,63 @@
+name: Helm update
+
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  verify-helm-changes:
+    name: Verify if AKO helm charts needs updates
+    runs-on: ubuntu-latest
+    env:
+      JWT_APP_ID: ${{ secrets.AKO_RELEASER_APP_ID }}
+      JWT_RSA_PEM_KEY_BASE64: ${{ secrets.AKO_RELEASER_RSA_KEY_BASE64 }}
+    steps:
+      - name: Checkout AKO repo
+        uses: actions/checkout@v4
+
+      - name: Install devbox
+        uses: jetify-com/devbox-install-action@v0.11.0
+        with:
+          enable-cache: 'true'
+
+      - name: Configure git
+        run: |
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
+
+      - name: Checkout Helm Repo
+        run: |
+          git clone https://github.com/mongodb/helm-charts.git ./helm-charts-cloned
+          ls -lah
+
+      - name: Verify if CRDs were changed
+        id: crd-check
+        env:
+          HELM_CRDS_PATH: "./helm-charts-cloned/charts/atlas-operator-crds/templates"
+        run: |
+          devbox run -- "make helm-upd-crds"
+
+      - name: Verify if RBAC were changed
+        id: rbac-check
+        env:
+          HELM_RBAC_FILE: "./helm-charts-cloned/charts/atlas-operator/rbac.yaml"
+        run: |
+          devbox run -- "make helm-upd-rbac"
+
+      - name: Create PR for helm-charts repo
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
you are using the regular github secret, which i guess worked in your tests. We do not need to use the gh app then.